mobile/src/reducers/user/user-info.reducer.ts

```diff
@@ -24,7 +24,7 @@ export const userInfoReducer: Reducer<UserInfoState> = (state = initState, actio
                 ...state,
                 employeeId: action.userEmployeeId
             };
-        case 'LOAD-USER-EMPLOYEE-PERMISSIONS-FINISHED':
+        case 'LOAD-USER-EMPLOYEE-PERMISSIONS-FINISHED': {
```

Comment:
why block {} ? doesn't make sense in JS context

Reply:
2ba853109522b786765bc854b0ffce8496172121